public/app/features/dashboard/dashgrid/PanelHeader/PanelHeaderTitleItems.tsx

```diff
@@ -0,0 +1,58 @@
+import { css } from '@emotion/css';
+import React from 'react';
+
+import { PanelData, GrafanaTheme2 } from '@grafana/data';
+import { Icon, ToolbarButton, useStyles2 } from '@grafana/ui';
+
+export interface Props {
+  alertState?: string;
+  data: PanelData;
+  panelId: number;
+}
+
+export function PanelHeaderTitleItems(props: Props) {
+  const { alertState, data } = props;
+  const styles = useStyles2(getStyles);
+
+  const alertStateItem = (
+    <ToolbarButton
+      icon={<Icon name={alertState === 'alerting' ? 'heart-break' : 'heart'} className="panel-alert-icon" />}
+      tooltip={`alerting is ${alertState}`}
+      className={styles.item}
+    />
+  );
+
+  const timeshift = (
+    <>
+      <ToolbarButton
```

Comment:
Timeshift info is not a button really (no action) , but I think long term it will be a button that opens a modal so maybe it's fine so long remove cursor and focus in the meantime

Reply:
I think it also should not be blue (that should be reserved for links), thoughts @Ijin08 ?